### PR TITLE
US pin-home06 documentation

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/Administration.rst [hunk 1]
```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
...
    returned if any.
+   
+   
+.. _ManagingNews:
+
+==============
+Managing News
+==============
+
+How to manage pinned block layouts
```
Comment (thomasdelhomenie) [inline docs/Administration.rst]:
```suggestion
How to display pinned News on a page
```

File: docs/Administration.rst [hunk 2]
```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
...
+-----------------------------------
+
+The home page CLV is a block that displays articles that have been pinned on the homepage by a user having the role publisher.
+The layout is not set by default, you have to edit the page layout then add a "Content by query" in the area in which you want to set the block.
```
Comment (thomasdelhomenie) [inline docs/Administration.rst]: I would replace this paragraph by something like:
"The pinned articles are not highlighted by default in the platform. In order to display them in a page, edit the page layout then add a "Content by query" application at the desired location."
And set a link on "pinned articles" to the user guide explaining how to pin an article.

File: docs/Administration.rst [hunk 3]
```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
...
+
+|image348| 
+
+|image349| 
+
+The first thing to do is to edit the portlet and choose the folder path.
```
Comment (thomasdelhomenie) [inline docs/Administration.rst]:
```suggestion
Once the application is added in the page, edit its settings and choose the folder path.
```

File: docs/Administration.rst [hunk 4]
```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
...
+
+|image350| 
+
+Then you have to choose the appriopriate template from "Display Settings" tab of "Edit Mode" : 
+
+- For the blog layout, select the following file "NewsPin.gtmpl".
```
Comment (thomasdelhomenie) [inline docs/Administration.rst]:
```suggestion
- For the blog layout, select "NewsPin.gtmpl".
```

File: docs/Administration.rst [hunk 5]
```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
...
+- For the latest News layout, select the following file "LatestNews.gtmpl".
```
Comment (thomasdelhomenie) [inline docs/Administration.rst]:
```suggestion
- For the latest News layout, select "LatestNews.gtmpl". 
```

File: docs/Administration.rst [hunk 6]
```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
...
+|image350| 
+
+Then you have to choose the appriopriate template from "Display Settings" tab of "Edit Mode" : 
+
+- For the blog layout, select the following file "NewsPin.gtmpl".
+- For the latest News layout, select the following file "LatestNews.gtmpl". 
+
+|image351| 
+
+Next, fill in the field "Header", this information will be displayed in the top right of the block.
```
Comment (thomasdelhomenie) [inline docs/Administration.rst]: top left ?

File: docs/Administration.rst [hunk 7]
```diff
@@ -4574,6 +4577,49 @@ The Service Management page is displayed.
...
+
+|image352| 
+
+From advanced tab, fill in the following query in the field "By query" :
+
+       *select * from exo:symlink where jcr:path like '/Application Data/News/Pinned/%' ORDER BY jcr:created DESC*
+
+Then select the appropriate page from News folder from the selector "Show in page"
```
Comment (thomasdelhomenie) [inline docs/Administration.rst]:
```suggestion
Then select the page *News* > *Details* from the selector "Show in page"
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/Administration.rst (hunk 2 above)
Comment (samehaidi) [reply docs/Administration.rst]: updated

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**